Stp.TestingApi/EnumMapper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Stp.TestingApi.Contracts;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Stp.TestingApi
+{
+    public static class EnumMapper
+    {
+        public static int EnumToInt(ExerciseComplexity theEnum)
+        {
+            return 0; 
+        }
+
+    }
```

Comment:
for what reason it created? ususal cast ((int)ExerciseComplexity.High) allows to get int value of the enum member.